docs/notebooks/fundamentals/queries-and-expressions.ipynb

```diff
@@ -54,35 +54,43 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "id": "af95c4aa-7905-44e9-b387-af512c347b68",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
```

Comment:
Best not to leave these extraneous messages floating around in user documentation